src/components/setup_meetings/RoomSetupMeeting.js

```diff
@@ -1,40 +1,59 @@
 "use-strict";
 import React, { Component } from "react";
-import { View, Text, FlatList, Dimensions, StyleSheet, SafeAreaView } from "react-native";
-import HeaderSetupMeeting from "../setup_meetings/HeaderSetupMeeting"
-
+import {
+  View,
+  Text,
+  FlatList,
+  Dimensions,
+  StyleSheet,
+  SafeAreaView
+} from "react-native";
+import HeaderSetupMeeting from "../setup_meetings/HeaderSetupMeeting";
```

Comment:
is there any way to use absolute routes instead of relative ones?

Reply:
Done